.env.example

```diff
@@ -52,3 +52,7 @@ ALCHEMY_API_KEY=
 
 # Discord
 GATSBY_JOIN_DISCORD_URL=https://dcl.gg/discord
+
+# Block Explorers
+ETHERSCAN_BASE_URL=https://etherscan.io/
```

Comment:
I believe it's not necessary to configure this at this level. If other project forks this repo they will probably still use the same block explorer for ETH and MATIC.